theano/tensor/nnet/corr.py

```diff
@@ -359,8 +377,8 @@ class CorrMM(BaseCorrMM):
         Set to `(1, 1)` to disable subsampling.
 
     """
-    def __init__(self, border_mode="valid", subsample=(1, 1)):
-        super(CorrMM, self).__init__(border_mode, subsample)
+    def __init__(self, border_mode="valid", subsample=(1, 1), openmp=None):
```

Comment:
In fact, we don't need that method for the 3 subclass. Can you just remove them?